lib/utils.ts

```diff
@@ -28,3 +35,40 @@ export const truncate = (str: string, noWords: number): string => {
     return str;
   }
 };
+
+const noPrefill = new Set(['file']);
+const singlePrefills = new Set(['select']);
+const iterablePrefills = new Set(['checkboxes', 'repeated']);
+
+/** Generate flexible initial values for a flexible schema */
+export const generateInitialValues = (
```

Comment:
A test for this functionality would be very helpful

Reply:
moved the tests over and fixed a bug we introduced when we refactored the types